etc/iso.c

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright (C) 2015 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#include "iso.h"
+#include <string.h>
+#include "kerneltypes.h"
+
+#define PVD_OFFSET 156
+#define ISO_BLOCK 2048
+
+struct directory_record {
+	uint8_t length_of_record;
+	uint8_t length_of_ext_record;
+	char loc_of_ext[8];
+	char data_length[8];
+	char rec_date_time[7];
+	char file_flags[1];
+	char file_flags_interleaved[1];
+	uint8_t interleave_gap_size;
+	char vol_seq_num[4];
+	uint8_t len_identifier;
+	char file_identifier[2048];
+};
+
+void print_n_tabs(int n);
```

Comment:
I don't think we are doing declarations for internal functions unless necessary.